backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/map/BpkPriceMapMarker.kt

```diff
@@ -70,9 +70,9 @@ fun BpkPriceMapMarker(
         tag = tag,
         title = title,
         visible = visible,
-        zIndex = if (status == BpkPriceMarkerStatus.Focused && zIndex == 0.0f) 1.0f else zIndex,
+        zIndex = if (status == BpkPriceMarkerStatus.Focused && zIndex == null) 1.0f else zIndex ?: 0.0f,
```

Comment:
there already was a change for the price marker to do this, but we missed it for the icon one. also aligned the nullability to be clearer about what the behaviour should be